hash/megadriv.xml

```diff
@@ -60,6 +60,51 @@ Known undumped pirates
   * Shizen Mahjong / Competition Mahjong (by Sachen, apparently dumped but hoarded)
   * Magic 7 Block - 百変七巧板
 
+Sega Game Toshokan games could be downloaded via the Mega Modem while a subscription was active. The existing dumps were either extracted from Sega's B-Club! game rental service or ripped from the Game no Kanzume compilations released on the Mega-CD.
```

Comment:
This line is definitely far too long – please wrap it at at a reasonable width (if in doubt, 80 columns should be OK).

Reply:
Ok, done.